### ISSUE TITLE
Sending e-mail failed if the SMTP server don't need to be authenticated.

### ISSUE THREAD

**Issue by hyunto** [scouter-project/scouter-plugin-server-alert-email#12]
Hello, Thank you for developing this plug-in.

When I configured this plugin, I can't send an e-mail via SMTP that don't need to be authenticated.
I installed the Scouter in private network and our SMTP don't need to be authenticated.

Although I already modified the source code and apply it, I think it would be nice to have an option to choose whether to authenticate with SMTP server or not.

I change the source code like this, and I can receive an e-mail with SMTP server that need no auth.
```java
boolean authEnabled = conf.getBoolean("ext_plugin_email_smtp_auth", true);

if (authEnabled) {
    email.setAuthenticator(new DefaultAuthenticator(username, password));
}
```

Thank you.

**Comment by nices96** [comment]
I applied your suggestion. Thank you.
Please check source code and released binary.

**Comment by hyunto** [comment]
Thank you for apply my suggestion.
I installed with new released binary(20180427) and e-mail has been sent successfully without smtp authentication.
I will close this issue.
